public/js/controllers/groupsCtrl.js

```diff
@@ -111,7 +111,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
   .controller("MemberModalCtrl", ['$scope', '$rootScope', 'Members', 'Shared', '$http', 'Notification', 'Groups',
     function($scope, $rootScope, Members, Shared, $http, Notification, Groups) {
       $scope.timestamp = function(timestamp){
-        return moment(timestamp).format('MM/DD/YYYY');
+        return moment(timestamp).format(User.user.preferences.dateFormat.toUpperCase());
```

Comment:
User is not passed into the controller, so either it would need to be added, or accessed via rootscope, like this:

```
return moment(timestamp).format($rootScope.User.user.preferences.dateFormat.toUpperCase());
```